### PR TITLE
Improve docs ahead of 6 June deadline

### PR DESCRIPTION
**Pull request by bravegrape**
The changes in this PR include:
 - recommendations from the CO transparency team
 - fixes for issues identified during the usability research sessions

Each commit covers one change. See commit message for a detailed explanation about the change.

### CODE REVIEW

File: source/documentation/publish_and_manage_data/add_data.md [hunk 1]
```diff
@@ -43,34 +43,11 @@ You must publish the data twice a year. Take a ‘snapshot’ of the roles on 31
...
-For junior roles at below SCS pay band 1 (or equivalent) you must publish the:
-
-* unit
-* senior role reported to
-* grade
-* payscale range
-* generic job title
-* number of posts in FTE
-* professional/occupational group
+You must fill in the [standard organogram template](https://ckan.publishing.service.gov.uk/publisher-files/Blank_Organogram_Template_latest.xls) with the details of your department's SCS 1 and 2 employees.
```
Comment (J-Lambo) [inline source/documentation/publish_and_manage_data/add_data.md]: Do you think we need to put Senior Civil Servant (SCS) in full?

File: source/documentation/publish_and_manage_data/add_data.md [hunk 2]
```diff
@@ -35,6 +35,8 @@ Some organisations, such as intelligence agencies, are exempt from publishing or
 
 Local authorities must publish senior staff data in a [different format](http://schemas.opendata.esd.org.uk/OrganisationStructure).
 
+For detailed policy guidance about publishing organograms, email the Cabinet Office transparency team at transparencydatateam@cabinetoffice.gov.uk.
```
Comment (J-Lambo) [inline source/documentation/publish_and_manage_data/add_data.md]: In line with style guide, I'd recommend making the email an active link

File: source/documentation/publish_and_manage_data/add_data.md (hunk 1 above)
Comment (bravegrape) [reply source/documentation/publish_and_manage_data/add_data.md]: Yup, I'll put it in full 👍

File: source/documentation/publish_and_manage_data/add_data.md (hunk 2 above)
Comment (bravegrape) [reply source/documentation/publish_and_manage_data/add_data.md]: Done 👌

File: source/documentation/publish_and_manage_data/add_data.md (hunk 1 above)
Comment (J-Lambo) [reply source/documentation/publish_and_manage_data/add_data.md]: Looks good 👍

File: source/documentation/publish_and_manage_data/add_data.md (hunk 2 above)
Comment (J-Lambo) [reply source/documentation/publish_and_manage_data/add_data.md]: 👍

**Pull request merged**